BossMod/Components/LineOfSightAOE.cs

```diff
@@ -73,7 +73,7 @@ public override void AddAIHints(BossModule module, int slot, Actor actor, PartyR
             if (BlockersImpassable)
             {
                 var blockers = Blockers.Select(b => ShapeDistance.Circle(b.Center, b.Radius)).ToArray();
-                hints.AddForbiddenZone(p => blockers.Min(b => b(p)));
+                hints.AddForbiddenZone(p => blockers.Min(b => b(p)), NextExplosion);
```

Comment:
This isn't right. The idea is that AOE happens in a few seconds, and blockers are already there - you have to move around them to reach safespot.